js/views/userPageVw.js

```diff
@@ -450,6 +456,7 @@ module.exports = Backbone.View.extend({
   renderItems: function (model) {
     "use strict";
     var self = this;
+    var select = document.getElementById('categories');
```

Comment:
var select = this.$el.find('.js-categories') give me:
Uncaught TypeError: select.appendChild is not a function